tests/golden/dev/appcat/appcat/apiserver/30_deployment.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   annotations:
-    metadata.appcat.vshn.io/enabled-service-hash: f1a27f82c3b5113e47030d7abaa6d596
+    metadata.appcat.vshn.io/enabled-service-hash: 3265bd6918e805528a293019e7e7d136
```

Comment:
@Kidswiss this seems to change even when the enabled vshn service list remains the same.... the `std.md5` is not working properly?

Reply:
`std.md5` should always produce the same hash for the same input. Maybe the `foldl` is missbehaving?

Reply:
This happened because I've disabled the `proxyFunction`. I've re-enabled it and the hash is now the same again